backend/app/helpers/orders.py

```
@@ -0,0 +1,145 @@
+from fastapi import HTTPException, status
+from sqlalchemy.orm import Session
+from app.models import Order, Product, Bill
```

Ruff (0.8.2)

`app.models.Order` imported but unused

Remove unused import

(F401)